Python-packages/covidcast-py/covidcast/plotting.py

```diff
@@ -74,39 +79,44 @@ def get_geo_df(data: pd.DataFrame,
                geo_type_col: str = "geo_type",
                join_type: str = "right") -> gpd.GeoDataFrame:
-    """Append polygons to a dataframe for a given geography and return a geoDF with this info.
+    """Augment a :py:func:`covidcast.signal` data frame with the shape of each geography.
 
     This method takes in a pandas DataFrame object and returns a GeoDataFrame object from the
-    `GeoPandas package <https://geopandas.org/>`__.
+    `GeoPandas package <https://geopandas.org/>`_.
 
     Shapefiles are 1:5,000,000 scale and sourced from the `2019 US Census Cartographic Boundary
     Files
-    <https://www.census.gov/geographies/mapping-files/time-series/geo/cartographic-boundary.html>`__
-
-    After detecting the geography type (either county or state) for the input, loads the
-    GeoDataFrame which contains state and geometry information from the Census for that geography
-    type. By default, it will take the input data (left side) and geo data (right side) and right
-    join them, so all states/counties will always be present regardless whether ``data`` contains
-    values for those locations. ``left``, ``outer``, and ``inner`` joins are also supported and
-    can be selected with the ``join_type`` argument.
-
-    For right joins on counties, all counties without a signal value will be given the value of
-    the megacounty (if present). Other joins will not use megacounties.
-
-    Returns the columns from ``data`` along with
-    `geometry` (polygon for plotting) and `state_fips` (FIPS code which will be used in the
-    plotting function to rearrange AK and HI) column. Coordinate system is GCS NAD83.
-
-    Default arguments for column names correspond to those used by :py:func:`covidcast.signal`.
-    Currently only supports counties and states.
+    <https://www.census.gov/geographies/mapping-files/time-series/geo/cartographic-boundary.html>`_.
+
+    After detecting the geography type (only county and state are currently
+    supported) of the input, builds a GeoDataFrame that contains state and
+    geometry information from the Census for that geography type. By default, it
+    will take the signal data (left side) and geo data (right side) and right
+    join them, so all states/counties will always be present regardless whether
+    ``data`` contains values for those locations. ``left``, ``outer``, and
+    ``inner`` joins are also supported and can be selected with the
+    ``join_type`` argument.
+
+    For right joins on counties, all counties without a signal value will be
+    given the value of the megacounty (if present). Other joins will not use
+    megacounties. See the `geographic coding documentation
+    <https://cmu-delphi.github.io/delphi-epidata/api/covidcast_geography.html>`_
+    for information about megacounties.
+
+    Default arguments for column names correspond to those used by
+    :py:func:`covidcast.signal`. Currently only supports counties and states.
```

Comment:
```suggestion
    :py:func:`covidcast.signal`.
```

This is already mentioned above, though if we wanted to put it more front and center, could move it to the second sentence of the docstring.

Reply:
Good point; I've rearranged the docstring a bit more and made other edits while I was at it